components/shared/canvas/webgl.rs

```diff
@@ -73,9 +73,15 @@ impl WebGLThreads {
     }
 
     /// Sends a exit message to close the WebGLThreads and release all WebGLContexts.
-    pub fn exit(&self) -> Result<(), &'static str> {
+    pub fn exit(
+        &self,
+        webgl_threads_sender: ipc_channel::ipc::IpcSender<()>,
+    ) -> Result<(), &'static str> {
         self.0
             .send(WebGLMsg::Exit)
+            .map_err(|_| "Failed to send Exit message")?;
+        webgl_threads_sender
```

Comment:
sorry I wasn't clear, this needs to go into the `Exit` message,

see for example https://github.com/servo/servo/blob/ca6169990ea8b429bbb8c11e4dcc9d8c6f59c7dd/components/net/resource_thread.rs#L375

Reply:
also we can follow the naming convention, so just `sender`.

Reply:
Yes I pushed the wrong changes